Algorithms/NumberTheory/SieveOfEratosthenes.cpp

```diff
@@ -0,0 +1,32 @@
+// Sieve Of Eratosthenes
+// Time Complexity O(nlog(log(n))) whereas following brute force will end you up with O(n*sqrt(n))
+#include <iostream>
+#include<math.h>
+using namespace std;
+
+int main(void){
+    int n;cin>>n;

```

Comment:
put it in next line.
edit all statements like n=4 to n = 4;
for loop write like this for (int i = 0; i < n; i ++)

Reply:
@lavishsaluja  edit your code with these changes.